network_plugin/__init__.py

```diff
@@ -145,7 +146,6 @@ def save_gateway_configuration(gateway, vca_client, ctx):
         return everything successfully finished
         raise NonRecoverableError - can't get task description
     """
-    wait_for_gateway(vca_client, gateway.get_name(), ctx)
```

Comment:
Looks as it will be regression in case several thread that tried to change Nat rules